sway/desktop/shaders/corner.frag

```diff
@@ -24,6 +24,11 @@ void main() {
     float smoothedAlphaInner = 1.0 - smoothstep(-1.0, 0.5, distance + half_thickness);
     gl_FragColor = mix(vec4(0), v_color, smoothedAlphaOuter - smoothedAlphaInner);
 
+    if ((v_color.a == 1.0 && gl_FragColor.a <= 0.5) || gl_FragColor.a <= 0.01) {
```

Comment:
This is tricky. This conditional statement seems to break AA, but if it's changed to simply

```
if (gl_FragColor.a <= 0.1)
```

We get the fringing issue :/

Reply:
Yeah, the fringing issue only occurs when the corner is extremely thin...

I wonder if we should inset the stencil by ~1px and render it before the borders